packages/dashboard-backend/src/devworkspace-client/services/cluster-info.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2018-2021 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+export const clusterConsoleUrl = process.env['OPENSHIFT_CONSOLE_URL'] || '';
```

Comment:
I am wondering how it is supposed to work if we do not have an environment variable OPENSHIFT_CONSOLE_URL

Reply:
The Applications menu won't be shown at all.